config/state_machine/configs/00_moj/20_sar/30_sar_trigger_workflow.yml

```diff
@@ -71,6 +71,7 @@
                 edit_case:
                   if: Workflows::Predicates#not_overturned?
                 update_closure:
+                  if: Workflows::Predicates#not_overturned?                  
```

Comment:
Test? Test! Or ... make predicated by a feature flag!

Reply:
Feature flag is the way forward!